contentcuration/contentcuration/frontend/shared/views/details/Details.vue

```diff
@@ -1,7 +1,7 @@
 <template>
```

Comment:
Moved from frontend/preview to better match how we're handling shared components